var/spack/repos/builtin/packages/swan/package.py

```diff
@@ -15,21 +17,40 @@ class Swan(MakefilePackage):
     the development of SWAN."""
 
     homepage = "http://swanmodel.sourceforge.net/"
```

Comment:
Since you're here, could you change this to use the `https` protocol instead of `http`?